did_doc/src/schema/service.rs

```diff
@@ -55,6 +52,13 @@ where
 
 #[derive(Debug)]
 pub struct ServiceBuilder<E> {
+    id: Uri,
+    service_endpoint: Url,
+    extra: E,
+}
+
+#[derive(Debug)]
+pub struct ServiceBuilderWithServiceType<E> {
```

Comment:
What is this for? Where is `service_type` specified? (can't find it in did:sov method or didcore)